src/amp-shadow.js

```diff
@@ -29,12 +29,13 @@ import {
   installBuiltins,
   installRuntimeServices,
 } from './runtime';
+import {cssText as ampDocCss} from '../build/ampdoc.css';
+import {cssText as ampElementCss} from '../build/ampelement.css';
```

Comment:
Maybe call these `ampDocCssText` and `ampElementCssText` for consistency?

Reply:
I personally prefer shorter names. Let me know if you feel strongly about it.

Reply:
I suggested this because we're still using the `cssText` variable name [elsewhere](https://github.com/ampproject/amphtml/pull/22458/files#diff-de6227d3a3cd747b3ad158c7471e4422R85) in the code.

I didn't want the information that these variables are actual css text to be lost in the variable rename.